tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml

```diff
@@ -0,0 +1,11 @@
+jobs:
+- job: Linux_CI_Dev
+  pool: Linux-CPU
+  steps:
+    - script: 'tools/ci_build/github/linux/run_dockerbuild.sh -o ubuntu16.04 -d cpu -r $(Build.BinariesDirectory) -x "--use_mklml"'
```

Comment:
not specific to this PR, but remind me what was the reasoning to have some options in the yml and some options in the build script? (e.g. --enable_mkldnn is in run_build.sh but --use_mklml is here)

Reply:
run_build.sh has the minimal set of the parameters.. we want to test builds with mklml, but dont want it when we packaging the releases (in order to reuse the script)